pkg/kubelet/status/status_manager.go

```diff
@@ -75,6 +76,8 @@ type manager struct {
 	// apiStatusVersions must only be accessed from the sync thread.
 	apiStatusVersions map[kubetypes.MirrorPodUID]uint64
 	podDeletionSafety PodDeletionSafetyProvider
+
+	podStartupLatencyTracker *util.PodStartupLatencyTracker
```

Comment:
I would prefer this be an interface, not a static type.

Reply:
Make the interface local to each package to avoid spider dependencies (like PodDeletionSafetyProvider)

Reply:
Make sense, thanks. I'll do that tomorrow.

Reply:
Added as a separate commit: https://github.com/kubernetes/kubernetes/pull/111930/commits/49a0902a8a9048f7389992dceea822c11658ad3b , please review